install_themes/action.yml

```diff
@@ -54,3 +58,54 @@ runs:
             ${{ inputs.container }} \
             composer update ${{ inputs.update_options }}
         fi
+
+    - name: Generate deprecation warning to summary page
+      shell: bash
+      run: |
+          echo "Deprecation Warning:"
+          echo "---------------------"
+          echo -e "\033[0;31m### This actions are deprecated and may not be supported ${{ inputs.install_theme}} \033[0m"
+          exit 1
+
+    - name: 'Output report'
```

Comment:
The output report step iterates over files which don't exist in  this context as the artifact containing them has not been checked out and processed. Lines 73-81 do something which does not make sense here.